nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -254,6 +254,17 @@ language governing permissions and limitations under the License. -->
             <artifactId>org.everit.json.schema</artifactId>
             <version>1.4.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.fluenda</groupId>
+            <artifactId>ParCEFone</artifactId>
```

Comment:
Sorry to be overly picky. This library uses javax.el-api which is licensed under CDDL 1.1, and its NOTICE has been copied to the NiFi overall NOTICE and the assembly's NOTICE. However it is not in the standard-nar's notice (nifi/nifi-nar-bundles/nifi-standard-bundle/nifi-standard-nar/src/main/resources/META-INF/NOTICE). This should have already been included here since the Jolt UI uses it, but since it has not been included, do you mind adding it? Just need to add the following line to the CDDL 1.1 section of the aforementioned file:

(CDDL 1.1) (GPL2 w/ CPE) Expression Language 2.2.4 API  (javax.el:javax.el-api:jar:2.2.4 - http://uel-spec.java.net)

Note that the Jolt UI uses el-api version 3.0.0 but your library is using 2.2.4. This will cause two different versions (in two different areas once unpacked, so no eviction or other issues), but as a NAR we would want either two entries (one for each version), or perhaps consider upgrading your library to use 3.0.0.